src/case_gather/services.py

```diff
@@ -61,9 +61,14 @@ def createCaseCreators(case_creator_names, case):
     a new one has to be created in the database
     """
     for case_creator_name in case_creator_names:
-        parliament_member = ParliamentMember.objects.get_or_create(
-            name=case_creator_name
-        )[0]
+        try:
+            parliament_member = ParliamentMember.objects.get(
+                name=case_creator_name
+            )
+        except ParliamentMember.DoesNotExist:
```

Comment:
could you do a log here?
I'm curious to see how often we'll run into this.
Would be nice to be able to dig into the logs later on

Reply:
There are a bunch of extra parliament members (varamenn) that sometimes come in and vote on issues, we would not have them in our system as we only have those directly elected

Reply:
Might be a good idea to log here, as we need to add some data manually when this happens (party, district, abbreviation) as not everything is available to the scraper